src/providers/local/index.ts

```diff
@@ -12,10 +14,18 @@ function createMiddleware (options) {
   const { IPX, IPXMiddleware } = require('ipx')
 
   const ipx = new IPX({
-    input: {
-      adapter: 'fs',
-      dir: options.dir
-    },
+    inputs: [
+      {
+        name: 'local',
+        adapter: 'fs',
+        dir: options.dir
+      },
+      {
+        name: 'remote',
```

Comment:
We should somehow guard this to only enable during `generate` to avoid service abuse

Reply:
We can deny all remote urls by default, and If the user wants to use a remote url she should add domains to module whitelist.
What do you think